src/utils/slicer.py

```diff
@@ -9,21 +9,16 @@
 class VTRSlicerWithText:
     def __init__(
         self,
-        window_size: int = 25,
-        stride: int = 10,
-        font: str = "fonts/NotoSans.ttf",
-        font_size: int = 15,
+        char2array: dict,
+        window_size: int = 32,
+        stride: int = 5,
```

Comment:
why did you change default values?

Reply:
Because these default values are in parser arguments: https://github.com/deepvk/vitrina/blob/8247cf637b717a4f48abe0e5b238dd6719962923/src/utils/config.py#L32
https://github.com/deepvk/vitrina/blob/8247cf637b717a4f48abe0e5b238dd6719962923/src/utils/config.py#L33
Should I need to return old values?

Reply:
Got it. Let's keep these values)